include/hyperelasticSolver.h

```diff
@@ -156,7 +159,9 @@ namespace Solid
                                                         Vector<double> &,
                                                         Vector<double> &);
 
-    void output_results(const unsigned int) const;
+    void output_results(const unsigned int) ;
```

Comment:
Since you made `strain` and `stress` mutable I guess this function can be `const` again?